src/main/java/org/mitre/synthea/engine/Components.java

```diff
@@ -101,7 +101,7 @@ public static class SampledData implements Serializable {
     public Double lowerLimit; // Lower limit of detection
     public Double upperLimit; // Upper limit of detection
     public List<String> attributes; // Person attributes containing TimeSeriesData objects
-    public transient List<TimeSeriesData> series; // List of actual series data collections
+    public List<TimeSeriesData> series; // List of actual series data collections
```

Comment:
Not sure why this was ever `transient` but it ended up making the export tests fail (because the TestHelper serializes all the patients).

Reply:
I'm not 100% sure that I fully understand the code, but I think this was marked as `transient` because it is copying information and could be recreated if deserialized. `setSeriesData` looks like it copies information from `Person.attributes` and puts it in `series`, which could be rerun on a rehydrated instance if necessary.

Reply:
Yes, you are correct in that it could be recreated if deserialized. But I see two issues with that:

1) the `Person.attributes` will likely have changed from the moment the `SampledData.series` was first initialized... meaning the value of the EKG variable in the year 2010 of the simulation won't be the same as the value of the EKG variable in the year 2015 of the simulation. We reuse patient attributes.

2) In the case of the export test failures, these `SampledData` objects are actually stored in `HealthRecord.Observation.value` so that during deserialization the `setSeriesData` method will not run...  which is why some of the export validations were breaking. In other words, Observations without values fail to validate.